installer/porter.yaml

```diff
@@ -56,18 +61,23 @@ credentials:
     description: Kubeconfig file for cluster where the operator should be installed
     path: /root/.kube/config
   - name: config.toml
-    description: Porter configuration file found in ~/.porter/config.toml
+    required: false
```

Comment:
You had it right for the installation service account parameter. In Porter if you want something to be optional, just set a default. In this context required doesn't do what you think: it is the list of required properties on the json schema for the parameter (used when the parameter is a json document).

So these should all be `default: ""` instead.